tests/rptest/tests/cluster_features_test.py

```diff
@@ -229,11 +229,13 @@ def test_license_upload_and_query(self):
 
         assert self.admin.put_license(license).status_code == 200
 
-        def obtain_license():
+        def obtain_configured_license():
             lic = self.admin.get_license()
-            return (lic is not None and lic['loaded'] is True, lic)
+            return (self.admin.is_sample_license(lic), lic)
```

Comment:
shouldn't this be `not sample_license`?

Reply:
No, we're waiting to observe the sample license (the one available in CI) inserted on line 230. Maybe you thought `is_sample_license` returns true for the evaluation period license? Happy to change to a better name if you have suggestions. I went with this one because the sample license available in CI is retrieved with the `sample_license()` method.

Reply:
Oops, yeah, naming confusion. No big deal